parsing/lexer.mll

```diff
@@ -751,6 +756,7 @@ and skip_hash_bang = parse
             set_pre_extra_docstrings pre_pos (List.rev a)
     in
     let rec loop lines docs lexbuf =
+      docs_initial_newline_state := NoLine;
```

Comment:
Could you lift that write out of the loop?

Reply:
Good point, lifted.